crates/router/src/connector/bitpay/transformers.rs

```diff
@@ -141,7 +141,7 @@ pub struct BitpayPaymentResponseData {
     pub redirect_url: Option<String>,
     pub refund_address_request_pending: Option<bool>,
     pub merchant_name: Option<String>,
```

Comment:
Do we need this field merchant_name? It is not being used anywhere

Reply:
`merchant_name` is not used anywhere. I will remove it